src/v2/BaseLoginRouter.js

```diff
@@ -149,6 +154,12 @@ export default Router.extend({
       uiSchemaTransformer,
       responseTransformer.bind({}, this.settings),
     )(idxResponse);
+
+    //On success flush sessionStorage, 'success' object in remediation signifies end of login flow
+    if(idxResponse.rawIdxState.success || idxResponse.rawIdxState.successWithInteractionCode) {
```

Comment:
do you still need this given the logic added to SuccessView?

Reply:
Here also wanted to be extra careful.. I think i should remove it.